android/src/main/java/com/magicleap/magicscript/scene/nodes/layouts/manager/GridLayoutManager.kt

```diff
@@ -60,10 +60,12 @@ class GridLayoutManager : SizedLayoutManager<GridLayoutParams>() {
             }
         }
 
-        val itemPadding = layoutParams.itemPadding
+        val defaultItemPadding = layoutParams.defaultItemPadding
+        val itemsPadding = layoutParams.itemsPadding?.paddings?.values
         val layoutSize = layoutParams.size
+
         if (layoutSize.x != WRAP_CONTENT_DIMENSION) {
-            val paddingHorizontal = itemPadding.left + itemPadding.right
+            val paddingHorizontal = getMaxHorizontalPadding(defaultItemPadding, itemsPadding)
```

Comment:
As I know, we should use `itemPadding` if both properties are specified (not max value).  The same applies to other layouts. From docs about `defaultItemPadding`.
> This is set for any new items that are added that don't explicitly specify padding.

Source https://github.com/magic-script/magic-script-components/blob/master/docs/components/GridLayout.md